pkgs/build-support/setup-hooks/make-symlinks-relative.sh

```diff
@@ -1,37 +1,35 @@
 # symlinks are often created in postFixup
 # don't use fixupOutputHooks, it is before postFixup
-postFixupHooks+=(_makeSymlinksRelativeInAllOutputs)
+if [[ -z "${dontRewriteSymlinks-}" ]]; then
+  postFixupHooks+=(_makeSymlinksRelative)
```

Comment:
Do you want to use https://github.com/NixOS/nixpkgs/pull/414284 for this now?

Reply:
Done